cmd/syncthing/gui_auth.go

```diff
@@ -128,6 +110,45 @@ func basicAuthAndSessionMiddleware(cookieName string, cfg config.GUIConfiguratio
 	})
 }
 
+func AuthSimple(username string, password string, configUser string, configPassword string) (bool, string) {
+
+    if username != configUser {
+        username = string(iso88591ToUTF8([]byte(username)))
```

Comment:
This "might be UTF-8, might be ISO-8859-1" business with the username and password is a property of HTTP basic auth and affects auth over LDAP as much as built in auth. Hence it should be cleared up on some higher level.